### PR TITLE
Switched to boto3 and added description field to each grant_row(recent AWS feature)

### PR DESCRIPTION
**Pull request by markocelan**
In order to use 'description' field one must use boto3. This PR is an attempt to do as little change to make it work.

### CODE REVIEW

File: ec2_security_groups_dumper/main.py [hunk 1]
```diff
@@ -54,18 +54,20 @@ def __init__(self,
             - rules_grants_group_id (unicode)
             - rules_grants_name (unicode)
             - rules_grants_cidr_ip (unicode)
+            - rules_description (unicode)
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: Can you update the docstring so the type match what's actually expected?

File: ec2_security_groups_dumper/main.py [hunk 2]
```diff
@@ -247,7 +258,8 @@ def csv(self):
                       'rules_to_port',
                       'rules_grants_group_id',
                       'rules_grants_name',
-                      'rules_grants_cidr_ip']
+                      'rules_grants_cidr_ip',
+                      'rules_description']
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: Can you add it to the docstring?

**Comment by markocelan** [comment]
I hope this does it.
@lra thank _you_ for the script 😄

**Comment by chris-x86-64** [comment]
Looks like this fixed Issue #15 .

**Pull request merged**